sync/command.py

```diff
@@ -303,13 +303,7 @@ def update_landing():
                 elif try_push.status == "complete" and not try_push.infra_fail:
                     update_landing()
                     if current_landing.latest_try_push == try_push:
-                        landing.try_push_complete(git_gecko,
-                                                  git_wpt,
-                                                  try_push,
-                                                  current_landing,
-                                                  allow_push=kwargs["push"],
-                                                  accept_failures=accept_failures,
-                                                  tasks=tasks)
+                        landing.push_to_gecko(git_gecko, git_wpt, current_landing)
```

Comment:
I'm not sure why we would skip `try_push_complete` here, maybe I'm missing something?

Reply:
If the try push is marked as complete we must have already run the `try_push_complete` logic on that try push without hitting an error. So we can immediately do the `push_to_gecko` step without rerunning the whole function on the try push.